model/model.pb.gw.go

```diff
@@ -865,7 +865,7 @@ func RegisterModelHandlerClient(ctx context.Context, mux *runtime.ServeMux, clie
 var (
 	pattern_Model_Liveness_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0}, []string{"__liveness"}, ""))
 
-	pattern_Model_Liveness_1 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1}, []string{"health", "pipeline"}, ""))
+	pattern_Model_Liveness_1 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1}, []string{"health", "model"}, ""))
```

Comment:
Please rename the assigned variable in a Go flavor camel-case.

Reply:
@pinglin variable such as pattern_Model_Liveness_1, right. It is generated file for our protobuf not our variables.

Reply:
Ooh yes right. This is an auto-gen repo. Nvm.